javascript/test/usage.js

```diff
@@ -0,0 +1,296 @@
+const assert = require('assert');
+const { sign, verify, extractKid, SignatureError } = require("..");
+
+// Use the same values as rust tests for cross-lang consistency assurance
```

Comment:
comment would not be required if the value was pulled from the same place for all tests regardless of language